### DISCUSSION TITLE
Encryption Support w/ Post-Quantum

### DISCUSSION THREAD

**Discussion by StephanSalas** [Seagate/cortx#1065] [Q&A]
Are plans in the development roadmap to support post-quantum encryption for cortx? What plans do we currently have for distributed-storage encryption?

[a comment by an automated account is not shown]

**Comment by justinzw** [comment]
@StephanSalas  thanks for asking this. We don't have plans to support post-quantum encryption. But I'd love to learn more about it. What type of distributed storage encryption would you like us to support and what use cases are you interested in using this for?

[a comment by an automated account is not shown]
